### ISSUE TITLE
[Low priority] Formatting for Python code

### ISSUE THREAD

**Issue by tcuongd** [facebook/prophet#1851]
This is definitely not a priority and is just a nice-to-have for contributors. We could set up [`black`](https://black.readthedocs.io/) and [`isort`](https://github.com/PyCQA/isort) for the python folder to handle code formatting and import ordering respectively.

We'd define settings for each in a `pyproject.toml` and then set them up to run during CI like this: https://github.com/psf/black/actions/runs/17913292/workflow

Thoughts? Happy to help with this :)

**Comment by bletham** [comment]
Thanks for the suggestion. I've definitely thought about this and am a big fan of black in the other projects that I work on. I actually tried running black here last Fall and the issue is that our formatting is really far out of spec and it changed practically every line. This completely destroys the blame, which makes debugging harder for me. At that time I decided that I preferred a useful blame to the code format benefits and didn't push the change. The other projects I work on that use black have used it from the beginning and it's been great; we might just be a little late here. that's still my current thinking on the matter but if there's an aspect that I'm missing I'd love to discuss it.
